.circleci/config.yml

```diff
@@ -155,6 +155,7 @@ jobs:
 
             bundle exec rake knapsack:rspec
             yarn test
+            yarn run lint
```

Comment:
I would consider breaking out all the lints as a separate job (sibling to `build`) so that way they can fail separately and clearly in the PR UI, and also more quickly

Reply:
So I think that woudl be:

```yaml
jobs:
  build:
    # .... leave it the same
  lints:
    steps:
      # .... lint steps here
# ...
workflows:
  release:
    - build
    - lints
    # the same
```

Reply:
Sounds good! I was following the precedent here set with running `slim-lint` as part of the "Run Tests" step. I assume in breaking out a separate lint job, we'd want to bring that one along as well?

Reply:
Yes please, bring slim lint over

Reply:
Added a new job in 664370a.